src/test/ajaxWrapper.test.js

```diff
@@ -39,6 +39,20 @@ describe("ajaxWrapper", function () {
       });
     });
 
+    it("defaults to GET when no method is supplied", function (done) {
+      ajaxWrapper({
+        url: config.apiURL
+      })
+      .success(function (response) {
+        expectAsync(function () {
+          expect(response.body.name).to.equal("Marathon");
+        }, done);
+      })
+      .error(function () {
```

Comment:
@pierlo-upitup 
Hmm, ok, I see the intention.
BTW if you forgot to "restore" the desired status, then you can call it a copy&paste-mistake.
Which is completely your bad. :)